README.md

```diff
@@ -9,108 +9,120 @@
 </p>
 <br/>
 
-## Access
-
-This Access Provider provisions a temporary Google Workspace Group assignment. You can use this in conjunction with SAML applications to allow users to request elevated access to SaaS applications.
-
-This Access Provider uses the following permission scopes:
-
-- `https://www.googleapis.com/auth/admin.directory.group`
-- `https://www.googleapis.com/auth/admin.directory.group.member`
-
 ## Getting started
 
-### Prerequisites
+The Google Workspace Access Provider provisions a temporary Google Workspace Group assignment. You can use this in conjunction with SAML applications to allow users to request elevated access to SaaS applications.
 
-To use this Access Provider you'll need to have [deployed Common Fate](https://docs.commonfate.io/common-fate/next/deploying-common-fate/deploying-common-fate). You'll also need to [download the `cf` CLI](https://docs.commonfate.io/common-fate/next/providers/setup).
+## Prerequisites
+
+To use this Access Provider you'll need to have [deployed Common Fate](https://docs.commonfate.io/common-fate/next/deploying-common-fate/deploying-common-fate) and [downloaded the `cf` CLI](https://docs.commonfate.io/common-fate/next/providers/setup).
 
 You will also need AWS credentials with the ability to deploy CloudFormation templates.
 
 ### 1. Generate credentials
```

Comment:
```suggestion
## 1. Generate credentials
```

very minor nitpick, it would be good to have the "Customer ID", "Admin Email" and so forth sections underneath the "Generate credentials" heading.

In your PR you've decreased Customer ID so that they're the same heading (h3 - `###`). I see why you've done this - you've added additional subsections to the "Credentials" heading.

I'm thinking we should de-indent the `Generate credentials` section and some of the other headers to match

```
# Getting Started - h1 (reduce by 1)

## 1. Generate Credentials - h2 (reduce by 1)

### Credentials - h3 (same as current)

#### Set up a service account (same as current)

## 2. Deploy the Access Provider - h2 (reduce by 1)
```